include/logging/log_backend.h

```diff
@@ -254,6 +254,15 @@ static inline const struct log_backend *log_backend_get(uint32_t idx)
 	return &__log_backends_start[idx];
 }
 
+/**
+ * @brief Get backend by name.
+ *
+ * @param[in] backend_name Name of the backend as defined by the LOG_BACKEND_DEFINE.
+ *
+ * @return Pointer to the backend.
```

Comment:
Pointer to the backend if found, NULL if not found.

Reply:
Addressed.